sdl_android_lib/src/com/smartdevicelink/protocol/WiProProtocol.java

```diff
@@ -19,8 +19,10 @@
 	byte _version = 1;
 	private final static String FailurePropagating_Msg = "Failure propagating ";
 
-	private static final int V1_V2_MTU_SIZE = 1500;
-	private static final int V3_V4_MTU_SIZE = 131072;
+	public static final int V1_V2_MTU_SIZE = 1500;
+	public static final int V3_V4_MTU_SIZE = 131072;
+	public static final int V1_HEADER_SIZE = 8;
+	public static final int V2_V4_HEADER_SIZE =12;

```

Comment:
This should just be V2_HEADER_SIZE. If we move to v5 without changing header size we'd have to change names, or leave it to be confusing.